### PR TITLE
Add organization admin crud events

### LINKED ISSUE #31421: Add Events for Organization Creation and Member Assignment
### Description

It would be nice to be able to listen for organization related events in a custom spi interface.

### Discussion

#30180

### Motivation

This feature will allow me to call my internal API in order to create that organization in a database and assign users to them

### Details

_No response_

### PR DESCRIPTION
**Pull request by sguilhen**
Closes #31421

### CODE REVIEW

**Review by edewit — APPROVED** [review]

**Review by vramik — CHANGES_REQUESTED** [review]
Changes LGTM, I've noticed that `Closes` keyword in both commit message and PR description references another pull request, could you please replace it with the issue number instead? Otherwise it's good to go. Thank you

**Review by vramik — APPROVED** [review]

**Review by ahus1 — APPROVED** [review]
Approved as of @vramik's review

File: services/src/main/java/org/keycloak/organization/admin/resource/OrganizationMemberResource.java [hunk 1]
```diff
@@ -97,6 +99,10 @@ public Response addMember(String id) {
 
         try {
             if (provider.addMember(organization, user)) {
+                adminEvent.operation(OperationType.CREATE).resource(ResourceType.ORGANIZATION_MEMBERSHIP)
```
Comment (Degoya-Studio) [inline services/src/main/java/org/keycloak/organization/admin/resource/OrganizationMemberResource.java:102]: would be nice to also add the ID of the member to the event. Same for remove. See https://github.com/keycloak/keycloak/issues/34013

https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/organization/admin/resource/OrganizationMemberResource.java

Comment (ahus1) [reply services/src/main/java/org/keycloak/organization/admin/resource/OrganizationMemberResource.java:102]: @Degoya-Studio - please create a new issue for this and reference it here.

Comment (ahus1) [reply services/src/main/java/org/keycloak/organization/admin/resource/OrganizationMemberResource.java:102]: ... or update the issue you referenced accordingly.

**Pull request merged**